src/googledrive.js

```diff
@@ -177,20 +229,31 @@ GoogleDrive.prototype = {
     }
   },
 
+  /**
+   * Create or update a file.
+   *
+   * @param {string} path - File path
+   * @param body - File content
+   * @param {string} contentType - File content-type
+   * @param {object} options
+   * @param {string} options.ifNoneMatch - Only create of update the file if the current ETag doesn't match this string
```

Comment:
It's perfectly possible to break lines in these descriptions. See other source files.